velero-plugins/buildconfig/restore.go

```diff
@@ -34,7 +34,11 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	itemMarshal, _ := json.Marshal(input.Item)
 	json.Unmarshal(itemMarshal, &buildconfig)
 
-	buildconfig, err := p.updateSecretsAndDockerRefs(buildconfig, input.Restore.Spec.NamespaceMapping)
+	buildconfigUnmodified := buildv1API.BuildConfig{}
+	itemMarshal, _ = json.Marshal(input.ItemFromBackup)
```

Comment:
just out of curiosity why it safe to ignore the errors here? I do see it ignored into unmodified code as well, so nothing special about this PR but just wondering..

Reply:
I actually have no idea here. We'd probably need to chech the git history to see the first use of this pattern in the plugins and maybe whoever wrote that might be able to tell us. I don't think it was me, but I can't confirm that off the top of my head.